packages/acs-ui-javascript-loaders/src/outboundCallCompositeLoader.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React from 'react';
+import { createRoot } from 'react-dom/client';
```

Comment:
Hmm this depends on the react version being used, i.e. will break if react 16 is used